docs/concepts/serving/executor/add-endpoints.md

```diff
@@ -187,6 +187,76 @@ with Deployment(uses=MyExec) as dep:
 {"__results__": {"my_executor/rep-0": {"internal_parameter": 20.0}}}
 ```
   
+
+## Streaming endpoints
+Executors can stream Documents individually rather than as a whole DocumentArray. 
+This is useful when you want to return Documents one by one and you want the client to immediately process Documents as 
+they arrive. This can be helpful for Generative AI use cases, where a Large Language Model is used to generate text 
+token by token and the client displays tokens as they arrive.
+Streaming endpoints receive one Document as input and yields one Document at a time.
+```{admonition} Note
+:class: note
+
+Streaming endpoints are only supported for HTTP protocol and for Deployment.
+```
+
+A streaming endpoint has the following signature:
+
+```python
+from jina import Executor, requests, Document, Deployment
```

Comment:
can u add this as supported part in `docarray v2` support, where u show an example with the new way?